install_config/install/advanced_install.adoc

```diff
@@ -355,10 +271,125 @@ configuration.
+
+----
+[OSEv3:vars]
+openshift_deployment_type=openshift-enterprise
+----
+endif::[]
+ifdef::openshift-origin[]
+Ensure the `deployment_type` parameter in your inventory file's `[OSEv3:vars]`
+section is set to `origin` to install the {product-title} variant:
+
+----
+[OSEv3:vars]
+openshift_deployment_type=origin
+----
+endif::[]
+
+
+[[configuring-host-variables]]
+=== Configuring Host Variables
+
+To assign environment variables to hosts during the Ansible installation, indicate
+the desired variables in the *_/etc/ansible/hosts_* file after the host entry in
+the *[masters]* or *[nodes]* sections. For example:
+
+----
+[masters]
+ec2-52-6-179-239.compute-1.amazonaws.com openshift_public_hostname=ose3-master.public.example.com
+----
+
+The following table describes variables for use with the Ansible installer that
+can be assigned to individual host entries:
+
+[[advanced-host-variables]]
+.Host Variables
+[options="header"]
+|===
+
+|Variable |Purpose
+
+|`*openshift_hostname*`
+|This variable overrides the internal cluster host name for the system. Use this
+when the system's default IP address does not resolve to the system host name.
+
+|`*openshift_public_hostname*`
+|This variable overrides the system's public host name. Use this for cloud
+installations, or for hosts on networks using a network address translation
+(NAT).
+
+|`*openshift_ip*`
+|This variable overrides the cluster internal IP address for the system. Use
+this when using an interface that is not configured with the default route.
+
+|`*openshift_public_ip*`
+|This variable overrides the system's public IP address. Use this for cloud
+installations, or for hosts on networks using a network address translation
+(NAT).
+
+|`*containerized*`
+|If set to *true*, containerized {product-title} services are run on the target master and
+node hosts instead of installed using RPM packages. If set to *false* or unset,
+the default RPM method is used. RHEL Atomic Host requires the containerized
+method, and is automatically selected for you based on the detection of the
+*_/run/ostree-booted_* file. See
+xref:../../install_config/install/rpm_vs_containerized.adoc#install-config-install-rpm-vs-containerized[Installing on Containerized Hosts] for more details.
+ifdef::openshift-enterprise[]
+Containerized installations are supported starting in {product-title} 3.1.1.
+endif::[]
+
+|`*openshift_node_labels*`
+|This variable adds labels to nodes during installation. See
+xref:configuring-node-host-labels[Configuring Node Host Labels] for more
+details.
+
+|`*openshift_node_kubelet_args*`
+|This variable is used to configure `kubeletArguments` on nodes, such as
+arguments used in xref:../../admin_guide/garbage_collection.adoc#admin-guide-garbage-collection[container and
+image garbage collection], and to
+xref:../../admin_guide/manage_nodes.adoc#configuring-node-resources[specify
+resources per node]. `kubeletArguments` are key value pairs that are passed
```

Comment:
Do we want a consistent format for key value pairs? 
A search on the doc site yields: key/value pair, key-value pair, key value pair.